### DISCUSSION TITLE
Music Library Tracks are showing up fine, but when in Album View - says "catalog/us/albums/.... not found"

### DISCUSSION THREAD

**Discussion by w00master** [music-assistant/support#2450] [Q&A] [answered]
Music Library Tracks are showing up fine, but when in Album View - says "catalog/us/albums/.... not found"
<img width="535" alt="Screenshot 2024-06-12 at 10 39 25 AM" src="https://github.com/music-assistant/hass-music-assistant/assets/37219704/8c173302-b8e5-4138-9fd0-1607f576bf31">

When searching for said track individually - plays fine and shows up.  It's only in album view does the individual tracks show up as "not found."  Strangely, not all tracks and albums do this.  Some albums display all the tracks in the library fine.  So not sure what's causing this issue.  I've tried reloading, syncing all the inputs, but nothing seems to resolve this.

This has been occurring since the last 3 updates.

**Comment by OzGav** [comment]
What provider is this coming from though?

**Comment by w00master** [comment]
It seems to be happening with the tracks that are specific to "my library"
<img width="869" alt="Screenshot 2024-06-12 at 12 04 48 PM" src="https://github.com/music-assistant/hass-music-assistant/assets/37219704/3efe5317-a039-43b4-8b1e-3e3da3dd0fb3">

In Album view, the Provider details pop up instantly as well as Other versions, but the "track" section is where it gets this error as you can see in the screenshot.  These items usually are specific to my library.

Sorry to add, this only seems to happen in Album view.

**Comment by w00master** [comment]
As a follow-up, here's a screencast video of what I'm talking about:
https://app.screencast.com/X8tAWUOK4OAil

You can see that the India Arie tracks show up fine individually in Track view and when I search.  However, looking at those same tracks in Album view, MA says "tracks not found."

Providers on my instance:
- Plex Music Server
- Apple Music
- Spotify
- YouTube Music

None of these appear to be having any issues individually.

**Comment by w00master** [comment, marked as answer]
Follow-up.  Your hint on Providers was the key. It's the new Apple Music integration that's causing the issue.  Once I disabled Apple Music, the Album view worked fine.  

As an aside, Apple Music has been working well for me - it's only in the Album View have I encountered any issues.  Regardless, this issue can be closed.  Thanks for all your help/hints!

**Reply by OzGav** [reply]
Sounds like this problem with Apple Music needs to be fixed though doesn’t it?

**Reply by w00master** [reply]
lol. Definitely. But I knew what I was going into when Apple Music still doesn’t have an official way of authenticating. 

Really love the app. And super appreciate all the help yall have provided. 

Side note: I think most of the providers and players are set and doing well for the most part. Are future plans to expand on exploration? I think this would be a huge move for you folks. It’s one of the reasons why Roon fans stay with Roon.

**Reply by OzGav** [reply]
On Roon I think that discussion is here https://github.com/orgs/music-assistant/discussions/1120

If you can easily recreate the Apple Music issue then please submit an issue report detailing all you have found and making it clear your exact setup with all the providers (in case there is an conflict)
